src/llhttp/http.ts

```diff
@@ -317,6 +317,7 @@ export class HTTP {
 
     n('req_http_start')
       .match('HTTP/', n('req_http_major'))
+      .match('RTSP/', n('req_http_major'))
```

Comment:
It is concerning that we conflate two different protocols here as this library is used in Node.js as well. Do you think we could use some option to generate support for RTSP on-demand. Maybe we could add `options` argument [here](https://github.com/nodejs/llhttp/blob/e4322448b9f411efa8e6f91dc6df4296cd4e425e/src/llhttp/http.ts#L140)?

Reply:
Perhaps, which is what ICE does in a different way one line below. However, ICE only appears to have one verb and RTSP has many more. TBH I'm not really familiar with EMCAScript/JavaScript/TypeScript at this point, so I'm not sure I'm the one to make a better change here. If you know of a way to select the correct verbs from a list rather than one at a time, I'm happy to try to implement that given a pointer.

Reply:
You're right... I didn't realize that ICE was treated on the same terms as HTTP. 

@nodejs/http could someone take a look at the comments in this PR and help decide whether adding implicit `RTSP` support would do any harm to Node?

Reply:
Thanks. Alternatively, selecting out the verbs for RTSP seems like the right behavior.. I just don't know how to express that in this language. It's the verbs I added plus OPTIONS.